otp.js

```diff
@@ -2,10 +2,12 @@ require('dotenv').config();
 const { createClient } = require('redis');
 const crypto = require('crypto');
 const Messente = require('messente_api');
+const geoip = require('geoip-lite');
```

Comment:
May be good to delete this and the code using it since we're not using IP to determine country